06_modules_lab/02.py

```diff
@@ -2,3 +2,14 @@
 and prints their sum.
 """
 
+import sys
+
+if len(sys.argv) > 2:
+    (name, strnum1, strnum2) = sys.argv
```

Comment:
Nobody cares about `name` here, so naming it can create confusion. Python has a special variable that holds values nobody cares about - the `_`. A more pythonic way would be:
`(_, strnum1, strnum2) = sys.argv`